lifecycleScripts/postinstall.js

```diff
@@ -30,7 +30,7 @@ module.exports = function install() {
     return Promise.resolve();
   }
 
-  return exec("node " + path.join(rootPath, "dist/nodegit.js"))
+  return exec("node \"" + path.join(rootPath, "dist/nodegit.js\""))
```

Comment:
I feel like `return exec("node \"" + path.join(rootPath, "dist/nodegit.js\""))` should instead be `return exec("node \"" + path.join(rootPath, "dist/nodegit.js") + "\"")` but this has been open for long enough. I'll fix that in a later PR.